improver_tests/acceptance/test_apply_emos_coefficients.py

```diff
@@ -310,25 +310,6 @@ def test_no_coefficients(tmp_path):
     )
 
 
-def test_wrong_coefficients(tmp_path):
```

Comment:
Again, why has this test case been removed?

Reply:
I've re-added this test and just caught the Iris exception that is being raised when the constraints, as used by `cli.create_constrained_inputcubelist_converter`, don't match the contents of the cube.